spec/support/features/post_helpers.rb

```diff
@@ -0,0 +1,18 @@
+module Features
+  module PostHelpers
+    def create_post(title, url)
+      visit(new_post_path)
+      fill_in('Title', with: title)
+      fill_in('Url', with: url)
+      click_button('Create Post')
+    end
+  end
+end
+
+RSpec.configure do |config|
+  config.include Features::PostHelpers, type: :feature
+end
```

Comment:
I'd move this `config.include` line to your `spec_helper.rb` file. That way it's easy to see all of the custom modules you've included in one place.